python-avd/pyavd/_eos_designs/schema/schema_fragments/network_ports.schema.yml

```diff
@@ -43,6 +43,7 @@ keys:
             The available template fields are:
               - `endpoint_type` - always set to `network_port`.
               - `endpoint` - content of the `endpoint` key if set.
+              - `port_channel_id`: The value from `port_channel.channel_id` for this network port if set.
```

Comment:
Strictly speaking this is just the port-channel ID, and not necessarily coming from that variable. We auto-generate the ID if not set. Just copy the description from the port-channels.

Reply:
done